src/app/pages/ProposalDetailsPage/index.tsx

```diff
@@ -87,10 +91,18 @@ export const ProposalDetailView: FC<{
       </dd>
 
       <dt>{t('networkProposal.create')}</dt>
-      <dd>{proposal.created_at}</dd>
+      <dd>
+        <Tooltip title={t('networkProposal.createTooltip')} placement={'top'}>
+          <Box>{proposal.created_at}</Box>
+        </Tooltip>
```

Comment:
Maybe we could consider adding the usual tooltip marker `(?)` here, to indicate that there is more info?